input/blog/posts/_current-newsletter-draft.md

```diff
@@ -18,7 +18,35 @@ Welcome to the .NET Foundation update for January & February. Every other month
 
 Each month Project Spotlight shines a light on an interesting open source project and its maintainers. In this iteration, .NET Foundation Marketing Committee member, Isaac Levin, speaks with ....
 
+### [NUnit](https://nunit.org)
 
+Many new releases are out at once, and work is starting on future major versions!
+
+#### [NUnit Framework 3.13](https://github.com/nunit/nunit/releases/tag/v3.13) and [3.13.1](https://github.com/nunit/nunit/releases/tag/v3.13.1)
+
+3.13 ships the first C# 8 Nullable Reference Types annotations! Almost all areas have been annotated. There are some important things to call out about this:
+
+- `Assert` methods don’t always throw exceptions when the assertion fails, due to the [Multiple Asserts](https://docs.nunit.org/articles/nunit/writing-tests/assertions/multiple-asserts.html) feature. This means that `Assert.NotNull(obj)` already didn’t mean that `obj` would never be null on the following line. `Assert.Catch` and the other `Assert` methods are the same way. We decided to not add nullability annotations unless they held true in every possible circumstance—a philosophy consistent with how .NET itself is annotated. This leads to the next point.
+
+- If you enable Nullable Reference Types in your test project, you will want to also add [NUnit.Analyzers 3.0](https://www.nuget.org/packages/NUnit.Analyzers/) or newer to your test project. Adding the analyzer is a great idea on its own, but it also specifically mitigates the problem in the previous point by suppressing C# compiler nullability warnings when we know it’s safe, when not in a Multiple Asserts context. This way you don’t have to use the C# null forgiveness operator (`!`) everywhere.
```

Comment:
I should update our release notes with this stuff. 👍🏻

Reply:
Sounds good. I reworded slightly.